app/models/visit.rb

```diff
@@ -1,4 +1,23 @@
 class Visit < ActiveRecord::Base
-  belongs_to :links
-  # attr_accessor :link_id, :user_id, :ip_address
+  belongs_to :user
+  belongs_to :link
+
+  scope :popular_links, lambda {
```

Comment:
Using a `do..end` block for the lambda required wrapping the block in parentheses before it would work. 
```
scope :popular_links, (lambda do 
...
end)
```
It looked awkward like that so I moved it to a class method.
Hope that is okay?